app/controllers/comments_controller.rb

```diff
@@ -7,23 +7,34 @@ class CommentsController < AuthenticatedController
   def create
```

Comment:
In this controller I miss the `track_created`, `track_updtaed` and `track_destroyed` calls.

Reply:
If I understand this correctly we are pushing `comment` related info through action cable.
I think the idea would be to make it generic and push `activity` related info.